betty/hashid.py

```diff
@@ -10,16 +10,25 @@
 
 import aiofiles
 
-_hasher = md5
+
```

Comment:
Extract this to a separate PR. Rename `content` to `key`. Pluralize `*content` to `*keys`

Reply:
Use "hash ID" instead of "ID"